docs/node-operators/get-started/setup-datanode.md

```diff
@@ -336,18 +351,180 @@ vega datanode start --home="YOUR_DATA_NODE_HOME_PATH"
+
+```shell
+No network history is available. Use the fetch command to fetch network history
+```
+
+In this case, it is possible to get the latest history segment ids from your network history bootstrap peers:
+
+```shell
+vega datanode network-history latest-history-segment-from-peers --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+This will return a list of peers you can fetch network history from:
+
+```text
+Most Recent History Segments:
+
+Peer:<peer-ip-and-api-port>, Swarm Key:<swarm-key>, Segment{from_height:75001  to_height:75300  history_segment_id:"some-segment-id"  previous_history_segment_id:"some-other-segment-id"}
+...
+```
+
+:::note
+It is possible that some data peers are behind and may not have the latest data. However, the `history_segment_id` should be the same across all nodes that are at the same height.
+:::
+
+
+To fetch a network history segment, run the command below. Use the ID of the segment you want (for example, the oldest) followed by the number of blocks prior to the segment's height that you want fetch. `2000` is used in the following example. This will result in all blocks from height N-2000 to N being retrieved.
+
+```shell
+vega datanode network-history fetch <segment-id-of-segment-at-height-N> 2000 --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+Once the network history segments have been downloaded, running: 
+
+```shell
+vega datanode network-history show --home="YOUR_DATA_NODE_HOME_PATH"
+``` 
+
+should display the network history you have:
+
+```text
+Available contiguous history spans:
+
+Contiguous history from block height XXXXX to XXXXX, from segment id: <first-segment-id> to <last-segment-id>
+
+
+Datanode contains no data
+```
+
+### Loading network history into data node
+Now that you have downloaded historical data you can load the history into the data node using:
 
 ```shell
-vega datanode fetch <segment-id-of-segment-at-height-5000> 2000
+vega datanode network-history load --home="YOUR_DATA_NODE_HOME_PATH"
+```
+
+:::note
+Data node must not be running when you use this command. If you attempt to use `network-history load` while data node is running, you will get an error:
+
+```text
+datanode must be shutdown before data can be loaded
+```
+:::
+
+You will be notified that the load command will force all existing connections to the data node database will be closed automatically and prompted if you want to continue. Enter `y` and data node will check how much history you have, tell you what it will load, and prompts you to confirm if you want to continue. Enter `y` again and the data node will load the history you have fetched.
+
+This process may take a very long time depending on how much history you have retrieved and are loading into the database. Some processes in the restoration can take a long time to complete and it may look like the process is not doing anything. Do not try to terminate the process early or you may end up with a corrupted data node database.
```

Comment:
should have fixed the corrupted data node problem, in current PR tbc

Reply:
I'll leave it in for now, better to err on the side of caution.